meta-sokol-flex-staging/dynamic-layers/openembedded-layer/recipes-support/pidgin/pidgin_%.bbappend

```diff
@@ -2,4 +2,4 @@
 # SPDX-License-Identifier: GPL-2.0
 # ---------------------------------------------------------------------------------------------------------------------
 
-RDEPENDS:finch += "python3-core"
+RDEPENDS:finch:feature-sokol-flex-staging += "python3-core"
```

Comment:
This should be `:append:feature-sokol-flex-staging` to mirror existing behavior, this version is entirely replacing `RDEPENDS:finch` rather than appending to it,  unless there's a reason we're changing the behavior here.